test/mailers/patient_mailer_test.rb

```diff
@@ -86,7 +88,7 @@ def teardown
   end
 
   %i[enrollment_sms_text_based enrollment_sms_weblink].each do |mthd|
-    test "#{mthd} no phone provided" do
```

Comment:
I think now that we want this to be backwards compatible we still want to include these old tests that just have a phone number rather than a messaging service SID.

Reply:
The name change on this test was a mistake -- as the "phone" that is referenced in the title is not the `ENV['TWILLIO_SENDING_NUMBER']` but rather the patient's `primary_telephone`